### PR TITLE
chore: Create blank GitHub issue template

### LINKED ISSUE #43: Blank issue template
## Create blank issue template

### PR DESCRIPTION
**Pull request by trillium**
Resolves #43 
# Overview
Provide initial blank issue template.

### CODE REVIEW

**Comment by trillium** [comment]
Rask mentioned that his work uses this template:

```md
# User story
As...
I want to...
So that...

# Context
Insert context here

# Tasks
Insert tasks here

# Acceptance criteria
Insert ACs here

# Useful Links
Add useful links here
```

**Comment by trillium** [comment]
Test pinging @addynoven

**Comment by addynoven** [comment]
name: Detailed Issue Template
about: Use this template for creating comprehensive and detailed issues.
title: "[Issue Type] Descriptive Issue Title"
labels: ['bug', 'enhancement', 'documentation']
assignees: ''
priority: 'medium'
milestone: ''
type: ''
due_date: ''

**Comment by trillium** [comment]
> name: Detailed Issue Template about: Use this template for creating comprehensive and detailed issues. title: "[Issue Type] Descriptive Issue Title" labels: ['bug', 'enhancement', 'documentation'] assignees: '' priority: 'medium' milestone: '' type: '' due_date: ''


Thanks for the feedback!

due_date, milestone, priority, type will be generally applied by our issue tracking system instead of directly within the issue template.

File: .github/ISSUE_TEMPLATE/blank-issue.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Blank Issue Template
+about: ''
+title: ''
+labels: ''
```
Comment (trillium) [inline .github/ISSUE_TEMPLATE/blank-issue.md:5]: Inline comment

Comment (addynoven) [reply .github/ISSUE_TEMPLATE/blank-issue.md:5]: okay let's go with GitHub issue template!

**Review by addynoven — APPROVED** [review]

**Comment by trillium** [comment]
@NickSramcik Can you take a look at this PR

**Review by alcpereira — APPROVED** [review]

**Review by NickSramcik — APPROVED** [review]

**Pull request merged**